### PR TITLE
chore:updates all examples to pull from the registry v1.0

### PR DESCRIPTION
**Pull request by g-awmalik**

### CODE REVIEW

File: examples/sap_hana_ha_simple/main.tf [hunk 1]
```diff
@@ -14,17 +14,9 @@
...
-# Build Hash: 5f4ef08feb4fed0e1eabc3bfc4b2d64d99001ae7
-#
 module "sap_hana_ha" {
-  # hardcoding the module for testing only
-  source = "../../modules/sap_hana_ha"
-
-  # reference the module by its registry URL and version as such:
-  # source = "terraform-google-modules/sap/google//modules/sap_hana_ha"
-  # version = "0.5.0"
+  source = "terraform-google-modules/sap/google//modules/sap_hana_ha"
```
Comment (bharathkkb) [inline examples/sap_hana_ha_simple/main.tf]: If we switch this in examples, tests will become invalid as they are no longer testing the code changes in the PR rather a static released version. We should use module swapper instead
example PR: https://github.com/GoogleCloudPlatform/terraform-google-secure-cicd/pull/33/files

**Review by bharathkkb — CHANGES_REQUESTED** [review]
Looks like this does not touch any modules, only examples/readme. We can mark this as a chore.

**Comment by comment-bot-dev** [comment]
@g-awmalik
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

File: examples/sap_hana_ha_simple/main.tf (hunk 1 above)
Comment (g-awmalik) [reply examples/sap_hana_ha_simple/main.tf]: thanks for the tip

**Comment by comment-bot-dev** [comment]
@g-awmalik
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

**Comment by comment-bot-dev** [comment]
@g-awmalik
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

**Review by bharathkkb — APPROVED** [review]

**Pull request merged**